### PR TITLE
POWRNG-61 - Export download button added

### PR DESCRIPTION
**Pull request by RafisSomeone**

### CODE REVIEW

File: src/app/components/export-button/export-button.component.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<button class="submit-btn position" color="primary" mat-raised-button type="submit" (click) = "downloadExport()">Export</button>
```
Comment (michalbarczyk) [inline src/app/components/export-button/export-button.component.html:1]: remove spaces in (click) assignment

File: src/app/components/export-button/export-button.component.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  ngOnInit(): void {
+  }
+
+  setQuestionnaireId(questionnaireId : number) : void{
+      this.questionnaireId = questionnaireId;
+  }
+
+  downloadExport(): void{
+    if (this.questionnaireId != undefined){
+    this.http.get(this.URL + '/' + this.questionnaireId).subscribe((data:ExcelExport) =>{ const blob = new Blob([this.stringToArrayBuffer(atob(data.base64StringFile))], { type: 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet' });
```
Comment (michalbarczyk) [inline src/app/components/export-button/export-button.component.ts:27]: fix indentations

File: src/app/interfaces/questionnaire/result/result.ts [hunk 3]
```diff
@@ -22,3 +22,8 @@ export interface TextResult extends Result {
 export enum QuestionType {
     LIKERT_SCALE, MULTIPLE_CHOICE, MULTIPLE_ANSWER, TEXT
 }
+
+export interface ExcelExport {
+    base64StringFile: string
+    fileName: string,
```
Comment (michalbarczyk) [inline src/app/interfaces/questionnaire/result/result.ts:28]: unnecessary comma

File: src/app/components/export-button/export-button.component.html (hunk 1 above)
Comment (RafisSomeone) [reply src/app/components/export-button/export-button.component.html:1]: Done.

File: src/app/components/export-button/export-button.component.ts (hunk 2 above)
Comment (RafisSomeone) [reply src/app/components/export-button/export-button.component.ts:27]: Done

File: src/app/interfaces/questionnaire/result/result.ts (hunk 3 above)
Comment (RafisSomeone) [reply src/app/interfaces/questionnaire/result/result.ts:28]: Deleted.

**Pull request merged**